In [0]:
!pip install turicreate

In [0]:
import turicreate

In [0]:
people = turicreate.SFrame('people_wiki.sframe')

In [87]:
people.head()

Columns:
	URI	str
	name	str
	text	str

Rows: 10

Data:
+-------------------------------+---------------------+
|              URI              |         name        |
+-------------------------------+---------------------+
| <http://dbpedia.org/resour... |    Digby Morrell    |
| <http://dbpedia.org/resour... |    Alfred J. Lewy   |
| <http://dbpedia.org/resour... |    Harpdog Brown    |
| <http://dbpedia.org/resour... | Franz Rottensteiner |
| <http://dbpedia.org/resour... |        G-Enka       |
| <http://dbpedia.org/resour... |    Sam Henderson    |
| <http://dbpedia.org/resour... |    Aaron LaCrate    |
| <http://dbpedia.org/resour... |   Trevor Ferguson   |
| <http://dbpedia.org/resour... |     Grant Nelson    |
| <http://dbpedia.org/resour... |     Cathy Caruth    |
+-------------------------------+---------------------+
+-------------------------------+
|              text             |
+-------------------------------+
| digby morrell born 10 octo... |
| alfred j lewy aka sandy le... |
| harpdog brown is a singer ... |
| franz rottensteiner born i... |
| henry krvits born 30 decem... |
| sam henderson born october... |
| aaron lacrate is an americ... |
| trevor ferguson aka john f... |
| grant nelson born 27 april... |
| cathy caruth born 1955 is ... |
+-------------------------------+
[10 rows x 3 columns]

### **Question:1 Download the Wiki People SFrame. Then open a new Jupyter notebook, import TuriCreate, and read the SFrame data.**

### **Answer: It is a dataset that contains  3 columns: URI,name and text. There are 59071 rows.**

In [88]:
len(people) # total number of rows in people Sframe

59071

### **Question:2 H​ow many rows are in the SFrame? (Do NOT use commas or periods.)**

### **Answer: 59071**

In [89]:
people[-1]['name'] #last person name

'Fawaz Damrah'

### **Question:3 Which name is in the last row?**

### **Answer: Fawaz Damrah**

In [93]:
brown=people[people['name']=='Harpdog Brown']
print(brown['text']) # or we can also use people[people['name']=='Harpdog Brown']['text']

['harpdog brown is a singer and harmonica player who has been active in canadas blues scene since 1982 hailing from vancouver he crossed tens of thousands of miles playing club dates and festivals in canada the northwestern united states and germanyover the years he has issued seven cds in 1995 his home is where the harp is won the muddy award for the best nw blues release from the cascade blues association in portland oregon as well that year it was nominated for a canadian juno for the best bluesgospel recording teamed up with graham guest on piano his cd naturally was voted 1 canadian blues album of 2010 by the blind lemon surveybrown tours extensively with his guitarist j arthur edmonds performing their electric mid1950s chicago blues either as a duo or with the full band while he is home he juggles a few combos working many venues big and small he also leads the harpdog brown band which is a gutsy traditional chicago blues band in 2014 they released what it is comprising mainly or

### **Question:4 Read the text column for Harpdog Brown. He was honored with:**

### **Solution: He was ownered with 'a life time membership to the hamilton blues society'.**

In [95]:
people.sort('text') #sorting th SFrame according to the text in ascending order.

Columns:
	URI	str
	name	str
	text	str

Rows: 59071

Data:
+-------------------------------+---------------------+
|              URI              |         name        |
+-------------------------------+---------------------+
| <http://dbpedia.org/resour... |     108 (artist)    |
| <http://dbpedia.org/resour... |        1.8.7        |
| <http://dbpedia.org/resour... | Diane Carlson Evans |
| <http://dbpedia.org/resour... |    Robert Godley    |
| <http://dbpedia.org/resour... |     Lisa Mullins    |
| <http://dbpedia.org/resour... |   Martijn Meeuwis   |
| <http://dbpedia.org/resour... |    8 Ball Aitken    |
| <http://dbpedia.org/resour... |  8th Arjia Rinpoche |
| <http://dbpedia.org/resour... |      Jim Gould      |
| <http://dbpedia.org/resour... |   David L. Holmes   |
+-------------------------------+---------------------+
+-------------------------------+
|              text             |
+-------------------------------+
| 108 born 1978 is an italia... |
| 187 is a pseudonym for jor... |
| 1lt diane carlson evans an... |
| 1robert godley b 7 july 19... |
| 2012 gracie award winner l... |
| 250px250pxmartijn meeuwis ... |
| 8 ball aitken born 8 june ... |
| 8th agya hotogtu lobsang t... |
| a canadian foreign affairs... |
| a church historian david l... |
+-------------------------------+
[59071 rows x 3 columns]
Note: Only the head of the SFrame is printed.
You can use print_rows(num_rows=m, num_columns=n) to print more rows and columns.

### **Question:5 Sort the SFrame according to the text column, in ascending order. What is the name entry in the first row?**

### **Answer: 108(artist)**